test/integration/crud/bulk.test.ts

```diff
@@ -1502,21 +1166,17 @@ describe('Bulk', function () {
     metadata: { requires: { topology: 'single', mongodb: '>2.5.4' } },
 
     test: function (done) {
-      var self = this;
-      var client = self.configuration.newClient(self.configuration.writeConcernMax(), {
-        maxPoolSize: 1
-      });
-
-      client.connect(function (err, client) {
-        var db = client.db(self.configuration.db);
-        var insertFirst = false;
-        var batchSize = 1000;
-        var collection = db.collection('batch_write_unordered_split_test');
-        var operation = collection.initializeUnorderedBulkOp(),
-          documents = [];
-
-        for (var i = 0; i < 10000; i++) {
-          var document = { name: 'bob' + i };
+      client.connect((err, client) => {
+        const db = client.db();
+        const insertFirst = false;
+        const batchSize = 1000;
+        const collection = db.collection('batch_write_unordered_split_test');
+        let operation = collection.initializeUnorderedBulkOp();
+        const documents = [];
+
+        let i = 0;
```

Comment:
Why did this variable get out of the initialization block in the loop?

Reply:
It's used outside the loop